login.php

```diff
@@ -26,10 +26,12 @@
       if($foundUser->password == $_POST['txtPassword']) {
         session_start();
         $_SESSION['sEmail'] = $sUserEmail;
-        $_SESSION['sName'] = $foundUser->name;
-        $_SESSION['sUserId'] = $foundUser->id;
+        $_SESSION['s  FirstName'] = $foundUser->firstname;
```

Comment:
Can you fix the whitespace and push again brudda?

Reply:
Sure man, there's actually still a lot of commented out things etc in it, I just thought lets leave it there for now since I've still got quite a bit to finish off on my end

Reply:
But hopefully it lets you work further as it is there now